src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -0,0 +1,50 @@
+@echo off 
+setlocal
+
+REM 
+REM Start WCF Self Hosted Service on local machine
+REM
+
+REM Build tools
+call BuildWCFSelfHostedService.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while building WCF Self hosted Service.
+    set __EXITCODE=%ERRORLEVEL% 
+    goto :done
+  )
+call BuildCertUtil.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while building the Certificate generator.
+    set __EXITCODE=%ERRORLEVEL%
+    goto :done
+  )
+
+
+REM Config Certs
+REM we need the direcotry to save the test.crl file. We are investigate a way to get rid of it
+REM
+md c:\wcftest
```

Comment:
Please make a note to find a way to remove these hard coded paths.   Not blocking for this merge.